modules/core/test/test_arithm.cpp

```diff
@@ -204,11 +203,39 @@ struct MulOp : public BaseElemWiseOp
     }
     void op(const vector<Mat>& src, Mat& dst, const Mat&)
     {
-        cv::multiply(src[0], src[1], dst, alpha);
+        int dtype = (flags & MIXED_TYPE) ? dst.type() : -1;
+        cv::multiply(src[0], src[1], dst, alpha, dtype);
     }
     void refop(const vector<Mat>& src, Mat& dst, const Mat&)
     {
-        cvtest::multiply(src[0], src[1], dst, alpha);
+        int dtype = (flags & MIXED_TYPE) ? dst.type() : -1;
+        cvtest::multiply(src[0], src[1], dst, alpha, dtype);
+    }
+    double getMaxErr(int depth)
+    {
+        return depth <= CV_32S ? 2 : depth < CV_64F ? 1e-5 : 1e-12;
```

Comment:
why 2?

Reply:
Actually, this part of code wasn't actually changed, it's inherited from previous commit
It's 2 everywhere where multiplication/division is performed
It can be safely dropped to 1 at least on my machine (will check it on CI)
I guess this threshold was tuned to actual accuracy level